extensions/wikia/Email/styles/main.css

```diff
@@ -1,6 +1,81 @@
-.content {
-	padding: 15px;
+/**
+ * This is the css file for the main.mustache template
+ * Note that it's a css file, not scss, so nesting and variables are not supported
+ * Only include styles that directly affect elements inside main.mustache
+ */
+
+body {
+	margin: 0;
+	padding: 0;
+}
+
+.body-wrap {
+	width: 100%;
+}
+
+.main-wrapper {
+	background-color: #E6EBF2;
+}
+
+.main-content {
+	background-color: #FFF;
```

Comment:
I don't hate you, I love coding conventions! It did look wonky with mix-matched cases in one file, so i made them all lower case. We don't have a rule per se but the [guidelines](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#hexformat) use lower case, so I changed it to that.